mesonbuild/interpreter.py

```diff
@@ -1534,6 +1558,8 @@ def func_files(self, node, args, kwargs):
     @noPosargs
     def func_declare_dependency(self, node, args, kwargs):
         version = kwargs.get('version', self.project_version)
+        if is_disabled(args, kwargs):
+            return DisableHolder()
```

Comment:
__[flake8]__
* [F821] undefined name 'DisableHolder'

<!-- Comment by [SideCI](https://sideci.com) -->

Reply:
Typo, should be `DisablerHolder`.